messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQServerService.java

```diff
@@ -267,8 +268,8 @@ public synchronized void start(final StartContext context) throws StartException
                 dbConfiguration.setDataSource(ds);
                 // inject the datasource into the PropertySQLProviderFactory to be able to determine the
                 // type of database for the datasource metadata
-                PropertySQLProviderFactory sqlProviderFactory = (PropertySQLProviderFactory)dbConfiguration.getSqlProviderFactory();
-                sqlProviderFactory.investigateDialect(ds);
```

Comment:
is the dialect also properly setup by the Artemis implementation?

Reply:
Yes